pjlib/src/pj/timer.c

```diff
@@ -489,7 +489,17 @@ static pj_status_t schedule_w_grp_lock(pj_timer_heap_t *ht,
     PJ_ASSERT_RETURN(entry->cb != NULL, PJ_EINVAL);
 
     /* Prevent same entry from being scheduled more than once */
-    PJ_ASSERT_RETURN(entry->_timer_id < 1, PJ_EINVALIDOP);
```

Comment:
I'll put it back - PJ_ASSERTs are effectively error logs (because we implemented soft assert behaviour), but now I actually think about it the `_RETURN` bit is still useful.